README.md

```diff
@@ -33,125 +44,110 @@ chmod +x /usr/local/bin/synse
 
 To save the binary to `/usr/local/bin/`. The binary can be saved to any location you wish.
 
-**NOTE**: We recommend that you save the binary with the name `synse` as it is easier to remember. All commands in this guide will use this name.
+> **NOTE**: We recommend that you save the binary with the name `synse` as it is easier to remember. All commands
+in this guide will use this name.
 
 You are now ready to run `synse` cli. To see a list of commands, type `synse -h`.
 
 ## Slightly Longer Way
 
 ### Building from Source
 
-Synse CLI is written in [Go](https://golang.org) and is provided as source. It can be compiled directly from source and built into a binary for any supported system.
+Synse CLI is written in [Go](https://golang.org) and is provided as source. It can be compiled directly from source
+and built into a binary for any supported system.
 
-**NOTE**: Windows is not supported at this time, due to incompatibilities in some utilized libraries. If there is enough interest this may be added in the future.
+> **NOTE**: Windows is not supported at this time, due to incompatibilities in some utilized libraries. If there is
+enough interest this may be added in the future.
 
-To get the code and compile it, simply clone this repository into your `GOPATH` and run the standard `go build` command. For example:
+To get the code and compile it, clone this repository into your `GOPATH`. Then, get the vendored dependencies
+with [dep](https://github.com/golang/dep) and build.
 
 ```shell
 git clone https://github.com/vapor-ware/synse-cli
 cd synse-cli
-go get
+dep ensure
```

Comment:
I just made some changes here in the i2c and rs485 makefiles (not yet up for PR) to do:
```dep ensure -update```
Because the latest sdk wasn't being picked up otherwise.